components/accordion/index.css

```
@@ -11,18 +11,154 @@ governing permissions and limitations under the License.
 */
 
 /* https://git.corp.adobe.com/Spectrum/spectrum-origins/pull/60 */
+
 .spectrum-Accordion {
-  /* Subtract the size of the border since it's on the item itself */
-  --spectrum-accordion-item-height-actual: calc(
-    var(--spectrum-accordion-item-height) -
-      var(--spectrum-accordion-item-border-size)
-  );
-  --spectrum-accordion-item-title-padding-y: var(
-    --spectrum-global-dimension-size-150
-  );
-  --spectrum-accordion-animation-duration: var(
-    --spectrum-global-animation-duration-100
-  );
+  --spectrum-accordion-item-height: var(--spectrum-component-height-200);
+  --spectrum-accordion-item-width: var(--spectrum-accordion-minimum-width);
+  --spectrum-accordion-disclosure-indicator-height: var(--spectrum-component-height-100);
+  --spectrum-accordion-disclosure-indicator-to-text-space: var( --spectrum-accordion-disclosure-indicator-to-text);
+  /*--spectrum-accordion-edge-to-disclosure-indicator-space: var(--spectrum-accordion-edge-to-disclosure-indicator);*/
+  /*--spectrum-accordion-edge-to-text-space: var(--spectrum-accordion-edge-to-text);*/
+  --spectrum-accordion-item-header-top-to-text-space: var(--spectrum-accordion-top-to-text-regular-medium);
+  --spectrum-accordion-item-header-bottom-to-text-space: var(--spectrum-accordion-bottom-to-text-regular-medium);
+  --spectrum-accordion-focus-indicator-gap: var(--spectrum-accordion-focus-indicator-gap);
+  --spectrum-accordion-focus-indicator-thickness: var(--spectrum-focus-indicator-thickness);
+  --spectrum-accordion-corner-radius: var(--spectrum-corner-radius-100);
+  --spectrum-accordion-item-content-area-top-to-content:var(--spectrum-accordion-content-area-top-to-content);
+  --spectrum-accordion-item-content-area-bottom-to-content: var(--spectrum-accordion-content-area-bottom-to-content);
+  --spectrum-accordion-component-edge-to-text: var(--spectrum-component-edge-to-text-75);
+  /* Text header */
+  --spectrum-accordion-item-header-font: var(--spectrum-sans-font-family-stack);
+  --spectrum-accordion-item-header-font-weight: var(--spectrum-bold-font-weight);
+  --spectrum-accordion-item-header-font-style: var(--spectrum-default-font-style);
+  --spectrum-accordion-item-header-font-size: var(--spectrum-font-size-300);
+  --spectrum-accordion-item-header-line-height: var(--spectrum-line-height-100);
+  &:lang(ja),
+  &:lang(zh),
+  &:lang(ko) {
+    --spectrum-accordion-item-header-line-height: var(--spectrum-cjk-line-height-100);
+  }
+  /* Text body */
+  --spectrum-accordion-item-content-font: var(--spectrum-body-sans-serif-font-family);
```

```suggestion
  --spectrum-accordion-item-content-font: var(--spectrum-sans-font-family-stack);
```
This font also needs the stack token. It is still rendering a serif font in Storybook.